models/vehicle.rb

```diff
@@ -110,8 +110,6 @@ def self.create(hash)
         hash[:unavailable_days].delete_if{ |index| !work_day_indices.include?(index.modulo(7)) }
       end
 
-      hash[:skills] = [[]] unless hash.has_key?(:skills) # If vehicle has no skills, it has the empty skillset
```

Comment:
This is needed. Otherwise we will have a single `[[]]` object if I am not mistaken.

Reply:
So you mean having a default value in vrp_input generates one single object ? We do have a different object_id when we use default value in vrp_input. Is that not enough? That would explain the problem I mentionned on test_using_two_solver

Reply:
I just saw @braktar's comment, it looks like grape fixed its single default object bug. So we can transfer the default to the grape you can ignore the above comment. 

However,  to fix mt-web case (i.e., `[]`) we need a  `hash[:skills] = [[]] if hash[:skills].empty?`  "conversion" somewhere. Before "everything" (i.e., before consistency and retrocomp checks) might be the best place.

Reply:
Is it okay for you as is i.e. in coerce_with ?

Reply:
That should be enough. We want not just the outer but the inner array to have different object_ids but if `hash[:vehicles][x][:skills].object_id` is different for different x's then `hash[:vehicles][x][:skills][0].object_id` would be different to. So I don't know.